docs/guides/05-introduction-to-styling.md

```diff
@@ -4,75 +4,55 @@ This guide walks through making a variety of visualizations for points, lines, a
 
 ### Supported Color Spaces
 
-* [HEX](https://carto.com/developers/carto-vl/reference/#cartoexpressionshex)
-* [RGB](https://carto.com/developers/carto-vl/reference/#cartoexpressionsrgb)
-* [RGBA](https://carto.com/developers/carto-vl/reference/#cartoexpressionsrgba)
-* [Named Colors](https://carto.com/developers/carto-vl/reference/#cartoexpressionsnamedcolor)
-* [HSV](https://carto.com/developers/carto-vl/reference/#cartoexpressionshsv)
-* [HSVA](https://carto.com/developers/carto-vl/reference/#cartoexpressionshsva)
-* [HSL](https://carto.com/developers/carto-vl/reference/#cartoexpressionshsl)
-* [HSLA](https://carto.com/developers/carto-vl/reference/#cartoexpressionshsla)
-* [CIELAB](https://carto.com/developers/carto-vl/reference/#cartoexpressionscielab)
+Supported color spaces are [HEX](https://carto.com/developers/carto-vl/reference/#cartoexpressionshex), [RGB](https://carto.com/developers/carto-vl/reference/#cartoexpressionsrgb), [RGBA](https://carto.com/developers/carto-vl/reference/#cartoexpressionsrgba), [Named Colors](https://carto.com/developers/carto-vl/reference/#cartoexpressionsnamedcolor), [HSV](https://carto.com/developers/carto-vl/reference/#cartoexpressionshsv), [HSVA](https://carto.com/developers/carto-vl/reference/#cartoexpressionshsva), [HSL](https://carto.com/developers/carto-vl/reference/#cartoexpressionshsl), [HSLA](https://carto.com/developers/carto-vl/reference/#cartoexpressionshsla), [CIELAB](https://carto.com/developers/carto-vl/reference/#cartoexpressionscielab).
 
 In addition to these [supported color spaces](http://cartovl-tabs.developers.carto-staging.com/developers/carto-vl/examples/#example-color-spaces), you can also use our [CARTOcolors](https://carto.com/carto-colors/) schemes directly.
 
-## Basic Styling Properties
+### Styling Properties
 
-* Color
-* Width
-* Stroke color
-* Stroke width
-* Resolution
-* Opacity
-
-### Color
+#### Color
 Use the `color` property to define the fill color of a point, line or polygon.
 
 ```
 color: #F24440
 ```
 
-### Width
+#### Width
 Use the `width` property to define a point size or line width.
 
 ```
 width: 5
 ```
 
-### Stroke Color
+#### Stroke Color
 Use the `strokeColor` property to define the color of a point or polygon stroke.
 
 ```
 strokeColor: #F24440
 ```
 
-### Stroke Width
+#### Stroke Width
 Use the `strokeWidth` property to define the size of a point or polygon stroke width.
 
 ```
 strokeWidth: 3
 ```
 
-### Resolution
+#### Resolution
 Use `resolution` to aggregate points into clusters.
 
 ```
 resolution: 5
 ```
 
-### Opacity
```

Comment:
Why did you remove this? @Jesus89 @IagoLast

Reply:
Opacity is not a `style property`. And `filter` was missing so I replaced it.

I just realized that we should move it to another place instead.

Reply:
Maybe not in the reference, but when you are trying to teach people about styling or designing a map, opacity is definitely part of it. Like the `a` in RGBA. We have to mention it.

Reply:
@Jesus89 the way we are defining opacity has always been a hard thing to explain (personally, I felt as though we should have a style property called `opacity` because it is very, very common as common as `color` and `width` in my opinion.

Since it isn't a "styling property" from the dev side but is indeed a styling property, where are you suggesting we put it?

If it is removed from a guide that walks through the basics of styling, it is likely that people will not find it.